.github/workflows/check_proofs.yml

```diff
@@ -0,0 +1,44 @@
+name: Check Proofs
+
+on: [push]
```

Comment:
Is push also for branches and pull requests?

Reply:
Yes - all pushes. The README badge only shows statuses for master though

Reply:
That is perfect :)